include/net/nrf_cloud.h

```diff
@@ -339,8 +348,9 @@ struct nrf_cloud_svc_info_fota {
 	uint8_t bootloader:1;
 	uint8_t modem:1;
```

Comment:
Maybe this should be renamed `modem_delta` to match up with the other renaming you have done? Or are you keeping it aligned with the API? Maybe a note here explaining the discrepancy would be a good idea?

Reply:
Honestly either way would be fine by me as long as there is a note explaining it, your choice

Reply:
actually, i'll just add a comment so i don't have to update projects that use this struct 😅